arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,200 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class AddFeaturesContingentValuesViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+        }
+    }
+    
+    let geodatabase: AGSGeodatabase!
+    /// The graphics overlay to add the feature to.
+    var graphicsOverlay = AGSGraphicsOverlay()
+    /// The geodatabase's feature table.
+    var featureTable: AGSArcGISFeatureTable?
+    /// The map point to add the feature to.
+    var mapPoint: AGSPoint?
+    
+    required init?(coder: NSCoder) {
+        let geodatabaseURL = Bundle.main.url(forResource: "ContingentValuesBirdNests", withExtension: "geodatabase")!
+        do {
+            // Create a temporary directory URL with subdirectories.
+            let temporaryDirectoryURL = FileManager.default.temporaryDirectory.appendingPathComponent(ProcessInfo().globallyUniqueString)
+            try? FileManager.default.createDirectory(at: temporaryDirectoryURL, withIntermediateDirectories: true)
+            // Create a temporary URL where the geodatabase URL can be copied to.
+            let temporaryGeodatabaseURL = temporaryDirectoryURL.appendingPathComponent("birdsNestGDB", isDirectory: false).appendingPathExtension("geodatabase")
+            // Copy the item to the temporary URL.
+            try FileManager.default.copyItem(at: geodatabaseURL, to: temporaryGeodatabaseURL)
+            // Create the geodatabase with the URL.
+            geodatabase = AGSGeodatabase(fileURL: temporaryGeodatabaseURL)
+        } catch {
+            print("Error setting up geodatabase: \(error)")
+            geodatabase = nil
+        }
+        
+        super.init(coder: coder)
+        
+        // Load the geodatabase.
+        geodatabase?.load { [weak self] (error) in
+            let result: Result<Void, Error>
+            if let error = error {
+                result = .failure(error)
+            } else {
+                result = .success(())
+            }
+            self?.geodatabaseDidLoad(with: result)
+        }
+    }
+    
+    deinit {
+        if let geodatabase = geodatabase {
+            geodatabase.close()
+            try? FileManager.default.removeItem(at: geodatabase.fileURL)
+        }
+    }
+    
+    /// Called in response to the geodatabase load operation completing.
+    func geodatabaseDidLoad(with result: Result<Void, Error>) {
+        switch result {
+        case .success:
+            // Get and load the first feature table in the geodatabase.
+            featureTable = geodatabase.geodatabaseFeatureTables[0] as AGSArcGISFeatureTable
+            featureTable?.load { [weak self] error in
```

Comment:
```suggestion
            let featureTable = geodatabase.geodatabaseFeatureTables[0] as AGSArcGISFeatureTable
            self.featureTable = featureTable
            featureTable.load { [weak self] error in
```
So it doesn't need to be conditionally unwrapped.